fedoracommunity/connectors/bodhiconnector.py

```diff
@@ -579,21 +585,25 @@ def query_active_releases(self, filters=None, **params):
                     if query.endswith('-testing'):
                         nvr = parse_build(release['nvr'])
                         row['testing_version'] = HTML.tag(
-                            'a', c='%(version)s-%(release)s' % nvr,
-                            href='%s/updates/%s' % (self._prod_url, nvr['nvr']))
+                            'a',
+                            c='%(version)s-%(release)s' % nvr,
+                            href=koji_build_url % nvr)
```

Comment:
- for all the build nvrs, point to koji.
- if there is also an associated update, then add a `(update)` link which points to bodhi.